UnitySDK/Assets/ML-Agents/Examples/Crawler/Scripts/CrawlerAcademy.cs

```diff
@@ -8,7 +8,7 @@ public override void InitializeAcademy()
         Monitor.verticalOffset = 1f;
         Physics.defaultSolverIterations = 12;
         Physics.defaultSolverVelocityIterations = 12;
-        Time.fixedDeltaTime = 0.01333f; // (75fps). default is .2 (60fps)
+        Time.fixedDeltaTime = 0.01333f; // (75fps). default is .02 (60fps)
```

Comment:
`<opinion>`
I do think we should avoid changing those in our environments.
`</opinion>`

Reply:
I don't know the history of them; since the scenes that override them are using ragdolls, I assume it was to improve stability. It might all change with the physx changes anyway.